lib/reel/websocket.rb

```diff
@@ -4,12 +4,13 @@
 
 module Reel
   class WebSocket
-    include Celluloid::Logger
+    extend Forwardable
     include ConnectionMixin
     include RequestMixin
 
-    attr_accessor :socket
-    
+    attr_reader :socket
```

Comment:
One afterthought @d-snp ... are there no cases where we'll need this to be `attr_writer` or `attr_accessor` as it was?